govcd/system.go

```diff
@@ -207,8 +207,15 @@ func CreateExternalNetwork(vcdClient *VCDClient, externalNetwork *types.External
 	externalNetwork.Configuration.FenceMode = "isolated"
 
 	// Return the task
-	return vcdClient.Client.ExecuteTaskRequest(externalNetHREF.String(), http.MethodPost,
+	task, err := vcdClient.Client.ExecuteTaskRequest(externalNetHREF.String(), http.MethodPost,
 		types.MimeExternalNetwork, "error instantiating a new ExternalNetwork: %s", externalNetwork)
+
+	// Real task in task array
+	if err == nil {
+		task.Task = task.Task.Tasks.Task[0]
```

Comment:
Please check that `task.Task.Tasks.Task[0]` exists before using it

Reply:
Added check